File: testjavabenchmark/domain/language/busuu/LearnFrench.java
package domain.language.busuu;

import infrastructure.socialmedia.SocialNetwork;

//Functional requirement 3.1.3
//Test case 98: Class domain.language.busuu.LearnFrench must use class infrastructure.socialmedia.SocialNetwork 
//Result: TRUE
public class LearnFrench {
	//FR5.5
	private SocialNetwork socialnetwork;
	
	public LearnFrench(){
		//FR5.2
		//System.out.println(socialnetwork.type);
		socialnetwork.type = "klant";
	}
}

File: testprojects/c#/domain/language/babbel/LearnFrench.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using CSharpBenchmark.infrastructure.asocialmedia;

namespace CSharpBenchmark.domain.language.babbel
{


    //Functional requirement 3.1.3
    //Test case 99: Class domain.language.babbel.LearnFrench must use class infrastructure.socialmedia.SocialNetwork 
    //Result: FALSE
    public class LearnFrench
    {
        //FR5.5
        private ASocialNetwork asocialnetwork;
        public LearnFrench()
        {
            //FR5.2
            //System.out.println(asocialnetwork.type);
            asocialnetwork.type = "klant";
        }
    }
}